### ISSUE TITLE
historical price graph not scaling correctly for timesteps without data

### ISSUE THREAD

**Issue by poliwop** [galacticcouncil/HydraDX-ui#1317]
Looking at for example the WETH price graph against USDT, we can see that timesteps at 16:00, 19:00 and 20:00 are equally spaced, probably because there were no trades at the 17:00 and 18:00 timesteps. The historical graph should scale the x-axis correctly in the case of no data.
![Screenshot from 2024-05-21 17-55-45](https://github.com/galacticcouncil/HydraDX-ui/assets/5674266/26402f13-9dc2-4cab-bc1c-1f2f8c307498)
![Screenshot from 2024-05-21 17-55-19](https://github.com/galacticcouncil/HydraDX-ui/assets/5674266/fbc8e6d1-73e4-42db-b8cb-abe1b53515d4)
![Screenshot from 2024-05-21 17-54-54](https://github.com/galacticcouncil/HydraDX-ui/assets/5674266/d2c2937d-b487-4d8e-8aad-1f2bd114d2ec)

**Comment by mrq1911** [comment]
yeah, you can see issue clearly when you look at the days in x axis, those should be equidistant @nohaapav

**Comment by nohaapav** [comment]
yeah, i guess this is default trade view settings ..
